src/devices/vdml_motors.cpp

```diff
@@ -259,6 +322,335 @@ std::int32_t Motor::set_reversed(const bool reverse) const {
 std::int32_t Motor::set_voltage_limit(const std::int32_t limit) const {
 	return motor_set_voltage_limit(_port, limit);
 }
+Motor_Group::Motor_Group(const std::initializer_list<Motor> motors) : _motors(motors), _motor_group_mutex(pros::Mutex()) {}
```

Comment:
TODO: add constructor that allows an initializer list full of numbers, and reverse numbers if the port number is negative.